README.md

```diff
@@ -117,13 +117,21 @@ Application secrets are not included in the code repository. File service relies
 | CLOUD_AZURE_STORAGE_BLOB_CONNECTION_STRING | Azure storage connection string                    |
 | CLOUD_AZURE_SERVICE_BUS_CONNECTION_STRING  | Azure service bus connection string                |
 
-###### Spring Local ENV Property Setting
+## Secrets
+
+Application secrets are not included in the code repository. Below are the instruction for each environment
+
+###### DEV
 
-Create **developer-local-properties.yaml** file under root of `resource` folder and set the application.yaml
+Create **developer-local-properties.yaml** file under root of `resource` folder and override the application.yaml
 placeholders.
 
 ```src/main/resources/developer-local-properties.yaml```
 
+###### PROD
+
+Secrets are configured as an environment variables on the deployment server.
```

Comment:
Remove `an` .
Secrets are configure as environment variables on the deployment server.